infrastructure/Scripts/storage-postdeployment.ps1

```diff
@@ -0,0 +1,15 @@
+param ($resourceGroup, $storageAccount)
+
```

Comment:
Doesn't the powershell script need to setup the azure environment with Get-AzAccount or this just automatically works because the deployment pipeline automatically loads the powershell environment?

Reply:
I referred to some examples [here ](https://github.com/Azure/azure-docs-powershell-samples/tree/master/storage)and don't find the `Connect-AzAccount` at the beginning of each script. I guess it operates within the context of the powershell environment (Portal or deployment pipeline)